src/Sentry/SentryOptions.cs

```diff
@@ -1292,17 +1293,321 @@ public SentryOptions()
         );
     }
 
-    internal void AddIntegration(ISdkIntegration integration)
```

Comment:
Is this related to the change from Extensions to ordinary class methods?

Reply:
Ah, I see... there was an equivalent public extension method 😵‍💫

Reply:
yes indeed. the method was already there on the extension class as public and then again here as internal 🤷